doc/configuration-v3_0.md

```diff
@@ -18,15 +18,15 @@ The Ignition configuration is a JSON document conforming to the following specif
     * **_httpTotal_** (integer) the time limit (in seconds) for the operation (connection, request, and response), including retries. 0 indicates no timeout. Default is 0.
   * **_security_** (object): options relating to network security.
     * **_tls_** (object): options relating to TLS when fetching resources over `https`.
-      * **_certificateAuthorities_** (list of objects): the list of additional certificate authorities (in addition to the system authorities) to be used for TLS verification when fetching over `https`.
+      * **_certificateAuthorities_** (list of objects): the list of additional certificate authorities (in addition to the system authorities) to be used for TLS verification when fetching over `https`. All certificate authorities must have a unique `source`.
         * **source** (string): the URL of the certificate (in PEM format). Supported schemes are `http`, `https`, `s3`, `tftp`, and [`data`][rfc2397]. Note: When using `http`, it is advisable to use the verification option to ensure the contents haven't been modified.
         * **_verification_** (object): options related to the verification of the certificate.
           * **_hash_** (string): the hash of the certificate, in the form `<type>-<value>` where type is sha512.
 * **_storage_** (object): describes the desired state of the system's storage devices.
-  * **_disks_** (list of objects): the list of disks to be configured and their options.
+  * **_disks_** (list of objects): the list of disks to be configured and their options. Every entry must have a unique `device`.
     * **device** (string): the absolute path to the device. Devices are typically referenced by the `/dev/disk/by-*` symlinks.
     * **_wipeTable_** (boolean): whether or not the partition tables shall be wiped. When true, the partition tables are erased before any further manipulation. Otherwise, the existing entries are left intact.
-    * **_partitions_** (list of objects): the list of partitions and their configuration for this particular disk.
+    * **_partitions_** (list of objects): the list of partitions and their configuration for this particular disk. Every partition must have a unique `number`, or if 0 is specified, a unique `label`.
```

Comment:
In the `number` field we have `If zero, use the next available partition slot`, do you think it's worthwhile to write it out here as well for consistency?

Reply:
I don't think it's necessary, but if you think it is, I can add it.

Reply:
I don't have a particularly strong opinion on changing it, I generally prefer to keep it consistent but I could see a case for this being a representation of the actual value that causes it to switch to being keyed off of `label`.

Not a blocker imo.

Reply:
I'll leave it for now unless someone else chimes in. The docs on `number` immediately follow it, so I'm not too worried.